web/styles/breakpoints.sass

```diff
@@ -125,7 +131,10 @@ $mobile-s:  320px
     &-mobile
       @media screen and (min-width: $mobile)
         display: none !important
-      
+    &-stories
```

Comment:
Maybe we should put `scale-show-stories` and `scale-hide-stories` in `home/index.sass` because it seems like a really specific style that we won't reuse.